lib/formatter/csv/annotation_for_csv.rb

```diff
@@ -125,7 +125,7 @@ def task_label(task_info)
       end
 
       def tool_label(task_info, drawn_item)
-        tool = task_info["tools"] && task_info["tools"][drawn_item["tool"]]
+        tool = task_info["tools"] && task_info["tools"][drawn_item.fetch("tool", 0)]
```

Comment:
Here instead of returning the 0 (first tool label) should we just not return a label?